README.adoc

```diff
@@ -542,6 +543,22 @@ image::https://lucid.app/publicSegments/view/43f2740c-2a7f-4b7f-909e-434a5bbe3fb
 
 See the link:{project_root}/parallel-consumer-examples/parallel-consumer-example-streams/src/main/java/io/confluent/parallelconsumer/examples/streams/StreamsApp.java[Kafka Streams example] project, and it's test.
 
+[[confluent-cloud]]
+=== Confluent Cloud
+
+. Provision your fully managed Kafka cluster in Confluent Cloud
+.. Sign up for https://www.confluent.io/confluent-cloud/tryfree/[Confluent Cloud], a fully-managed Apache Kafka service.
+.. After you log in to Confluent Cloud, click on `Add cloud environment` and name the environment `learn-kafka`.
+Using a new environment keeps your learning resources separate from your other Confluent Cloud resources.
+.. From the `Billing & payment` section in the Menu, apply the promo code `CC100KTS` to receive an additional $100 free usage on Confluent Cloud (https://www.confluent.io/confluent-cloud-promo-disclaimer[details]).
+.. Click on https://confluent.cloud/learn[LEARN] and follow the instructions to launch a Kafka cluster and to enable Schema Registry.
+. Access the client configuration settings
+.. From the Confluent Cloud Console, navigate to your Kafka cluster.
+From the `Clients` view, get the connection information customized to your cluster (select `Java`).
+.. Create new credentials for your Kafka cluster, and then Confluent Cloud will show a configuration block with your new credentials automatically populated (make sure `show API keys` is checked).
+.. Use these settings presented to https://docs.confluent.io/clients-kafka-java/current/overview.html[configure your clients].
+. Use these clients for steps outlined in the <<common_preparation>> section.
```

Comment:
@astubbs Even from the `common_preparation`, it's not clear which files need to be modified and how the settings should be configured.

Reply:
There is no file to be modified - the examples just mock out the clients. The step above links to the instructions on how to configure and instantiate the clients. The common prep shows how to pass those clients into PC. Make sense?

Reply:
👍 got it.  Thanks!